specs/management.yml

```diff
@@ -6023,6 +6023,101 @@ paths:
       name: id
       in: path
       required: true
+  /identities/{id}/trace:
+    put:
+      security:
+      - ztSession: []
+      description: |
+        Allows an admin to enable/disable data flow tracing for an identity
+      tags:
+      - Identity
+      - Tracing
+      summary: Enable/disable data flow tracing for an identity
+      operationId: updateIdentityTracing
+      parameters:
+      - description: A traceSpec object
+        name: traceSpec
+        in: body
+        required: true
+        schema:
+          $ref: '#/definitions/traceSpec'
+      responses:
+        "200":
+          description: Returns the document that represents the trace state
+          schema:
+            $ref: '#/definitions/traceDetailEnvelope'
```

Comment:
For the rest of the API a `PUT` to a document URL returns an `emptyResponse` from the shared responses.